src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -512,4 +512,26 @@ public static boolean isName(String str) {
 
         return isName;
     }
+
+    /**
+     * Returns true if the value arg is either null, or empty
+     * or contains only whitespace characters.
+     * More efficient that calling (string).trim().length() == 0
+     * or (string).trim().isEmpty()
+     * because it does not create temporary string whereas trim() does
+     * @param value A string to check
+     * @return true if the value does not contain any character except whitespace.
```

Comment:
if I pass null string as a parameter it will also return true. This is a util method, it will be used by the future contributors. The documentation and the name should be correct, otherwise this method may cause problems in the future. Its name already describes that the method does more than one action. 

@romani 
The method which does exactly the same check is called 'isBlank' in Apache Commons, but its name puzzles me as I always need to check the implementation to make sure that null and empty strings are checked. Thus, I cannot get along with the name of the method (isOnlyWhitespacesOrEmpty). 

My proposals:
1) isNullOrBlank
2) isNullOrEmptyOrWhitespacesOnly

As it is a static util method it is not a good idea to give it a long name, however, the client of the class API should know that null strings are also taken into account.